Localisation/NumberToWords/SpanishNumberToWordsConverter.ts

```diff
@@ -0,0 +1,106 @@
+module Humanizer.Localisation.NumberToWords {
+    "use strict";
+
+    interface ExceptionsDictionary {
```

Comment:
This interface isn't used any where. I see that an earlier version of your code used it but it's not needed now. Could you please remove it?